### PR TITLE
CODEOWNERS: maintainers for `dockerTools`

### PR DESCRIPTION
**Pull request by FRidh**
###### Motivation for this change

The `dockerTools` currently don't have any documented maintainers. Maintainers for functions go into the `CODEOWNERS` file.

Would you want to be listed as `dockerTools` maintainer? 

cc recent contributors that made multiple/larger changes @roberth @utdemir @zowoq @nlewo @adisbladis @nagisa @grahamc

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -206,6 +206,12 @@
 /nixos/tests/cri-o.nix                       @NixOS/podman @zowoq
 /nixos/tests/podman.nix                      @NixOS/podman @zowoq
 
+# Docker tools
+/pkgs/build-support/docker
+/nixos/tests/docker-tools-overlay.nix
+/nixos/tests/docker-tools.nix
+/doc/builders/images/dockertools.xml
```
Comment (roberth) [inline .github/CODEOWNERS:213]: :+1: 

```suggestion
/pkgs/build-support/docker                   @roberth
/nixos/tests/docker-tools-overlay.nix        @roberth
/nixos/tests/docker-tools.nix                @roberth
/doc/builders/images/dockertools.xml         @roberth
```

**Comment by FRidh** [comment]
Thank you @roberth . Anyone else can of course open a PR adding themselves as well.

**Comment by nlewo** [comment]
I'm currently no longer building Docker images with Nix, but i would be happy to open a PR if i work on this topic again.

**Comment by utdemir** [comment]
Hey, sorry I did not see this. Yes, I probably should be the code owner for `streamLayeredImage` stuff. I created a PR adding myself: https://github.com/NixOS/nixpkgs/pull/103175

**Pull request merged**